test/bats/vault.bats

```diff
@@ -79,13 +79,13 @@ EOF
   export K8S_CACERT="$(kubectl config view --raw \
     -o go-template="{{ range .clusters }}{{ if eq .name \"${CLUSTER_NAME}\" }}{{ index .cluster \"certificate-authority-data\" }}{{ end }}{{ end }}" | base64 --decode)"
 
-  run kubectl exec -it $VAULT_POD -- vault write auth/kubernetes/config \
+  run kubectl exec $VAULT_POD -- vault write auth/kubernetes/config \
     kubernetes_host="${K8S_HOST}" \
     kubernetes_ca_cert="${K8S_CACERT}" \
     token_reviewer_jwt="${TR_ACCOUNT_TOKEN}"
   assert_success
 
-  run kubectl exec -it $VAULT_POD -- vault policy write example-readonly -<<EOF
+  run kubectl exec -ti $VAULT_POD -- vault policy write example-readonly -<<EOF
```

Comment:
do we still need `-it` here?

Reply:
yeah, this is still required. Without `-it` the multiline command fails based on testing locally.